docs/business/organizations.rst

```diff
@@ -19,6 +19,14 @@ The best way to think about this relationship is:
 
 *Owners* will create *Teams* to assign permissions to all *Members*.
 
+Team Types
+~~~~~~~~~~
+
+You can create two types of Teams:
+
+* **Admins** -- These teams have full access to administer the projects in the team. They are allowed to change all of the settings, set notifications, and everything else under the **Admin** tab.
```

Comment:
'everything else' could include a verb. something along the lines of 'perform any action under ...' is clearer language